apps/artwork/components/auction/templates/bid.jade

```diff
@@ -12,16 +12,9 @@ if auction.is_open && !artwork.is_sold && !auction.is_live_open
       //- Rendered client-side
     if user
       .artwork-auction__bid-form__max-bid
-        if me && me.lot_standing.is_highest_bidder
```

Comment:
Just noticed that this line is currently throwing an error in production (for when `me` exists but `me.lot_standing` is null). So it'll be good to get this out there :D